enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -96,6 +136,18 @@ This is an example of an OKD cluster hosting solution where several organization
 #### As an OKD admin, I need to forward my audit logs to a secure SIEM that meets government regulations
 
 This is often required for industries such as the US public sector, healthcare or financials. The logs will be forwarded to a government approved SIEM through secure means (mutual TLS).
+#### As an administrator of a dedicated cluster, I need to allow dedicated-admins to configure the destination of Application logs but prevent them from modifying the destination of Infrastructure logs.
+
+Separate `forwarder` objects can be created with different access controls for
+independent groups.
```

Comment:
i'd like to see more detail/examples about how this works.

Reply:
ok i found the examples below but i think this misses the mark on at least one scenario:

"as a dedicated-admin i want to configure the forwarding of all applicaiton logs to a destination of my choice".

the per namespace forwarding resource doesn't cover this because i don't want to have to create a forwarding config resource in every single namespace (which my users might mess with also).

and the global forwarding resource doesn't cover it because as a dedicated-admin i'm not allowed to edit that (because it also controls where audit+infra logs go)

Reply:
You've hit the nail on the thumb, this is why we've backed down for GA. Here's what I think but there will be discussion:

I think everyone agrees on:
1. A cluster-scoped ClusterLogForwarder, singleton for GA. 
2. A namespace-scoped LogForwarder post GA, one per namespace (at least) allowed.

The controversy begins when I claim that all our problems can be solved (ahem) by making both of these things proper multi-instance objects with no magic singletons or restrictions.
I'm a big fan of the principle that if you are being forced to generalize it is better to go from 1 to N than 1 to 2.

1.  This solves the  "2 kinds of admin" problem (actually the "N kinds of admin") All kinds of admin (but only admins) are allowed to create a ClusterLogForwarder, but they can't mess with each others. If one ClusterLogForwarder needs to be blessed with special powers then we can tie it to a magic name as a singleton and control access that way. 

2. NamespaceLogForwarders are for projects without admin rights, so one per namespace you say? But what if there are multiple roles (developers, operators yada yada) who want to have control over "their own" log forwarding - exactly like the admin problem? 

Once we have a NamespaceForwarder we *already* have to solve all the problems of supporting multiple forwarders on the same set of log streams. Why not re-use that work to solve customer problems also?